### PR TITLE
Updates for getting started page

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Includes: 
1. Sample page link update
2. Template link update
3. Description for TFX / XGboost corrected 
4. Added a build your own sample for TFX pipeline.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3038)

### CODE REVIEW

**Comment by SinaChavoshi** [comment]
image

File: frontend/src/pages/GettingStarted.tsx [hunk 1]
```diff
@@ -28,26 +28,32 @@ const options = {
...
 
+<br/>
 
 **Tutorials** - Learn pipeline concepts by following a tutorial.
 
   * [Data passing in python components](#/pipelines) - Shows how to pass data between python components. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
   * [DSL - Control structures](#/pipelines) - Shows how to use conditional execution and exit handlers. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures)
 
-  You can find additional tutorials and samples [here]()
+  Want to learn more? find additional tutorials and samples [here](https://github.com/kubeflow/pipelines/tree/master/samples)
```
Comment (joeliedtke) [inline frontend/src/pages/GettingStarted.tsx]: - Remove the leading white-space. Markdown is currently interpreting this line as part of the list.
- Suggested revision(making the link more descriptive):

  **Want to learn more?** [Learn from sample and tutorial pipelines.](https://github.com/kubeflow/pipelines/tree/master/samples)

- Consider changing the URL in the link to use the samples and tutorials in the docs:
  https://www.kubeflow.org/docs/pipelines/tutorials/

File: frontend/src/pages/GettingStarted.tsx [hunk 2]
```diff
@@ -28,26 +28,32 @@ const options = {
 };
 
 const PAGE_CONTENT_MD = `
+<br/>
+
 ## Build your own pipeline
 
-Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb) (Alpha)
+Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/ai-platform/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fstorage.googleapis.com%252Fjiyongjung-public%252Ftfx%252Fnotebook%252Fcreate_a_pipeline_using_a_template.ipynb) (Alpha)
```
[the remark this answers is not in the document]
Comment (joeliedtke) [reply frontend/src/pages/GettingStarted.tsx]: TFX -> TensorFlow Extended (TFX)

File: frontend/src/pages/GettingStarted.tsx [hunk 3]
```diff
@@ -28,26 +28,32 @@ const options = {
...
+<br/>
 
+## Demos and Tutorials
 This section contains demo and tutorial pipelines.
 
 **Demos** - Try an end-to-end demonstration pipeline.
 
-  * [TFX pipeline demo](#/pipelines) - A trainer that does end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss)
-  * [XGBoost Pipeline](#/pipelines) - Example pipeline that does classification with model analysis based on a public taxi cab BigQuery dataset. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
+  * [TFX pipeline demo](#/pipelines) - Classification pipeline with model analysis based on a public taxi cab BigQuery dataset. To build your own [start here!](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb)
```
Comment (joeliedtke) [inline frontend/src/pages/GettingStarted.tsx]: Suggested revision: "Classification pipeline with model analysis based on a public taxi cab BigQuery dataset."

Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips.

File: frontend/src/pages/GettingStarted.tsx [hunk 4]
```diff
@@ -28,26 +28,32 @@ const options = {
...
   * [Data passing in python components](#/pipelines) - Shows how to pass data between python components. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
   * [DSL - Control structures](#/pipelines) - Shows how to use conditional execution and exit handlers. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/DSL%20-%20Control%20structures)
 
-  You can find additional tutorials and samples [here]()
+  Want to learn more? find additional tutorials and samples [here](https://github.com/kubeflow/pipelines/tree/master/samples)
+
+<br/>
 
 ### Additional resources and documentation
   * [TFX Landing page](https://www.tensorflow.org/tfx)
```
Comment (joeliedtke) [inline frontend/src/pages/GettingStarted.tsx]: TFX Landing page -> TFX documentation

Comment (joeliedtke) [reply frontend/src/pages/GettingStarted.tsx]: Kubeflow Pipeline open source documentation -> Kubeflow Pipelines documentation

File: frontend/src/pages/GettingStarted.tsx [hunk 5]
```diff
@@ -28,26 +28,32 @@ const options = {
...
 ## Build your own pipeline
 
-Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb) (Alpha)
+Build an end-to-end ML pipeline with TFX  [Start Here](https://console.cloud.google.com/ai-platform/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fstorage.googleapis.com%252Fjiyongjung-public%252Ftfx%252Fnotebook%252Fcreate_a_pipeline_using_a_template.ipynb) (Alpha)
 
-## Demos and Tutorials
+<br/>
 
+## Demos and Tutorials
 This section contains demo and tutorial pipelines.
```
[the remark this answers is not in the document]
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: >Suggestion: demo -> demonstration
Done
> ...would be nice to reinforce that demo=demonstration in this sentence.
would it help to leave the second bullet point as **Demos** ...

File: frontend/src/pages/GettingStarted.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: > TFX -> TensorFlow Extended (TFX)

Done

> Can "Start Here" be styled as a button?

I can turn it to a button though it feels somewhat out of place, how about bold or underline ?

File: frontend/src/pages/GettingStarted.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: >"Unified DSL" or a "TFX pipeline"?

I think we should change the pipeline name to match "TFX pipeline" , added the change to this PR. 

>Does the link to (#/pipelines) go to the pipelines index page or to this specific pipeline?

Pipeline index page, we can not link it to this specific pipeline for now however there is a feature request tracking that work post beta

> Suggested revision for: " To build your own [start here!]" -> "Learn how to get started with TFX pipeline templates."

Done with a small caveat this points to a TFX notebook not templates, I changed it to "Learn how to get started with TFX pipeline."

File: frontend/src/pages/GettingStarted.tsx [hunk 6]
```diff
@@ -28,26 +28,32 @@ const options = {
...
 
 **Demos** - Try an end-to-end demonstration pipeline.
 
-  * [TFX pipeline demo](#/pipelines) - A trainer that does end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss)
-  * [XGBoost Pipeline](#/pipelines) - Example pipeline that does classification with model analysis based on a public taxi cab BigQuery dataset. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
+  * [TFX pipeline demo](#/pipelines) - Classification pipeline with model analysis based on a public taxi cab BigQuery dataset. To build your own [start here!](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252F0.1.40%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb)
+  * [XGBoost Pipeline demo](#/pipelines) - An end-to-end distributed training for XGBoost models. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm)
```
[the remark this answers is not in the document]
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: Done.

File: frontend/src/pages/GettingStarted.tsx (hunk 1 above)
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: >Remove the leading white-space. Markdown is currently interpreting this line as part of the list.

Done

>Suggested revision(making the link more descriptive):

Done

>Consider changing the URL in the link to use the samples and tutorials in the docs:
https://www.kubeflow.org/docs/pipelines/tutorials/

Done

File: frontend/src/pages/GettingStarted.tsx (hunk 4 above)
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: >TFX Landing page -> TFX documentation

Done

>Kubeflow Pipeline open source documentation -> Kubeflow Pipelines documentation

Done

File: frontend/src/pages/GettingStarted.tsx (hunk 3 above)
Comment (SinaChavoshi) [reply frontend/src/pages/GettingStarted.tsx]: done

**Comment by SinaChavoshi** [comment]
image

**Comment by rmgogogo** [comment]
/lgtm

@Yuan, your PR seems can be reverted as this PR handled all.

**Comment by Bobgy** [comment]
@rmgogogo FYI, this is me.
I don't think there are any conflicts with my changes. LGTM

**Comment by rmgogogo** [comment]
> @rmgogogo FYI, this is me.
> I don't think there are any conflicts with my changes. LGTM

sorry, didn't expect it will at another one named Yuan.

**Pull request merged**